game/static/game/css/landing.css

```
@@ -158,74 +158,174 @@ body {
+
+.btn-secondary-ghost:hover {
+    background-color: rgba(30, 41, 59, 0.65);
+    border-color: rgba(255, 255, 255, 0.3);
+    color: #ffffff;
+    transform: translateY(-1px);
 }
 
-.btn-secondary:hover {
-    background-color: rgba(255, 255, 255, 0.05);
-    border-color: rgba(255, 255, 255, 0.2);
+@keyframes microEnginePulse {
```

Stylelint (17.11.1)

[error] Expected keyframe name "microEnginePulse" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)